app/views/manuals/_content_for_head_tag.html.erb

```diff
@@ -2,6 +2,15 @@
   <% if description.present? %>
     <meta name='description' content='<%= description %>' />
   <% end %>
+
+  <% if @content_store_section %>
```

Comment:
Would be better not to use instance variables in partials, instead passed them through as local variables of this partial.

- http://www.carlosramireziii.com/stop-using-instance-variables-in-partials.html
- https://forum.upcase.com/t/best-practice-dont-use-instance-variables-in-partials/2004